### PR TITLE
Problem: no api to convert native events to logs

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- add event converters

Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: x/evm/keeper/state_transition.go [hunk 1]
```diff
@@ -415,6 +415,9 @@ func (k *Keeper) ApplyMessageWithConfig(ctx sdk.Context,
 		vmError = vmErr.Error()
 	}
 
+	// convert native events to ethereum logs
+	convertNativeEvents(stateDB, k.eventConverters)
```
Comment (mmsqe) [inline x/evm/keeper/state_transition.go]: should we mv inside the commit

Comment (yihuang) [reply x/evm/keeper/state_transition.go]: not necessary I think

File: CHANGELOG.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - (cli) [#242](https://github.com/crypto-org-chain/ethermint/pull/242) Integrate tendermint bootstrap cmd.
 - (cli) [#246](https://github.com/crypto-org-chain/ethermint/pull/246) Call app.Close to cleanup resource on graceful shutdown.
 * (cli) [#288](https://github.com/crypto-org-chain/ethermint/pull/288) make abci handshake shutdown gracefully.
+- (evm) [#]() Add native event converter APIs.
```
Comment (mmsqe) [inline CHANGELOG.md]:
```suggestion
- (evm) [#343](https://github.com/crypto-org-chain/ethermint/pull/343) Add native event converter APIs.
```

**Review by mmsqe — APPROVED** [review]

File: x/evm/keeper/state_transition.go (hunk 1 above)
Comment (yihuang) [reply x/evm/keeper/state_transition.go]: Even if we don't commit in eth_call, we still want to see the logs

**Pull request merged**